### ISSUE TITLE
Make the school nodes visible again

### ISSUE THREAD

**Issue by CarlosBrys** [gravitystorm/openstreetmap-carto#3914]
I have noticed that in the render they have removed the visualization of the nodes labeled as schools, making them disappear from the visualization.

In many places, a school is a very important reference point for the community and it is necessary to show them to guide people.

In addition, many schools of different levels share the same building, making it very difficult to label an area with the data of all the schools that share it.

I suggest you reconsider the decision to stop rendering the nodes labeled as schools, and make them visible again, because it is very important for many communities.

**Comment by HolgerJeromin** [comment]
Duplicate of #3876
Already fixed, but not deployed

**Comment by matkoniecz** [comment]
Please comment with link to an example if that issue is still present.

**Comment by diegokrein** [comment]
Hello, For Example

https://www.openstreetmap.org/?mlat=-25.25006&mlon=-58.05432#map=18/-25.25006/-58.05432

In this location there are three schools of different levels of education. They share the building and operate at different times.

**Comment by matkoniecz** [comment]
In this case node is blocked by other label.

**Comment by CarlosBrys** [comment]
> In this case node is blocked by other label.

Could you be more explicit?
We would like to know what is the other label that blocks the visualization of the node, and thus be able to fix it.
In some cases only the name of the school is shown, but the node is not visible.
Thank you.

**Comment by matkoniecz** [comment]
>  In some cases only the name of the school is shown, but the node is not visible.

See #120 - schools have no icons and so far never had. There is an open issue proposing to add them.

**Comment by jragusa** [comment]
I'm wondering if it would be efficient to display schools mapped as node with a node and not an icon.

Here is an example where school is mapped as node next to other nodes creating inconsistent rendering because all of these features are located in the same building.
https://www.openstreetmap.org/node/6696772432
![school_node](https://user-images.githubusercontent.com/9897203/67228291-3a4dc800-f439-11e9-9861-1cc3bd06d6bb.png)

However, I noticed that `amenity=childcare` are rendered with a similar colour with a node:
https://www.openstreetmap.org/node/6159331797
![childcare](https://user-images.githubusercontent.com/9897203/67228507-a9c3b780-f439-11e9-8e7a-cf9338e6fd74.png)

Note that the same problem also concerns `amenity=kindergarten`:
https://www.openstreetmap.org/node/2499954008 (next to the first example)
![kindergarten](https://user-images.githubusercontent.com/9897203/67228677-16d74d00-f43a-11e9-9cbd-73d90825b2ba.png)
